pkg/controllers/cluster/resource/resource.go

```diff
@@ -510,3 +546,14 @@ func stringOrDefault(str, def string) string {
 	}
 	return def
 }
+
+func GetIpFromService(svc *corev1.Service, c *scyllav1.ScyllaCluster) (string, error) {
+	ip := svc.Spec.ClusterIP
+	if c.Spec.Network.HostNetworking {
+		var ok bool
+		if ip, ok = svc.ObjectMeta.Labels[naming.IpLabel]; !ok {
+			return "", errors.Errorf("%s label not found on member service %s", naming.IpLabel, svc.Name)
```

Comment:
I would suggest to add a bit more information in error message like ' (is corresponding pod running ?)'

Reply:
svc.Name is exactly the same than the pod.Name so from this logs it looks quite easy to find the appropriate pod.